### PR TITLE
Reconcile the `core` and `napari` READMEs

### PR DESCRIPTION
**Pull request by willGraham01**
Reconciles the information found in the two separate READMEs into a single file.

There was a large amount of redundancy in the text, as the `napari` plugin essentially just copied the bulk of its citation and description from the `core` package README.

Note that there is still a reference to `brainreg-segment` in the README here, which will need to be updated to `brainglobe-segmentation`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,174 @@
...
+[![Tests](https://img.shields.io/github/workflow/status/brainglobe/brainreg/tests)](
+    https://github.com/brainglobe/brainreg/actions)
+[![codecov](https://codecov.io/gh/brainglobe/brainreg/branch/master/graph/badge.svg?token=FbPgwBIGnd)](https://codecov.io/gh/brainglobe/brainreg)
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/python/black)
+
+# brainreg
+
+brainreg is an update to [amap](https://github.com/SainsburyWellcomeCentre/amap_python) (which is itself a port
+of the [original Java software](https://www.nature.com/articles/ncomms11879)) to include multiple registration backends, and to support the many atlases provided by [bg-atlasapi](https://github.com/brainglobe/bg-atlasapi).
+It also comes with an optional [napari plugin](https://github.com/brainglobe/brainreg-napari) if you'd rather use brainreg with through graphical interface, however this interface is slightly limited compared to the command line tool.
```
Comment (adamltyson) [inline README.md:14]: This is only slightly nowadays I think. Better to remove it, considering we want to promote the plugin as the main entry point.
```suggestion
It also comes with an optional [napari plugin](https://github.com/brainglobe/brainreg-napari) if you'd rather use brainreg with through graphical interface.
```

File: README.md [hunk 2]
```diff
@@ -1 +1,174 @@
...
+
+Documentation for both the command-line tool and graphical interface can be found [here](https://brainglobe.info/documentation/brainreg/index.html).
+This software is at a very early stage, and was written with our data in mind.
```
Comment (adamltyson) [inline README.md:17]: This is out of date
```suggestion

```

File: README.md [hunk 3]
```diff
@@ -1 +1,174 @@
...
+Over time we hope to support other data types/formats.
```
Comment (adamltyson) [inline README.md:18]: Out of date. 
```suggestion
Over time we hope to support other data types/formats.
```

File: README.md [hunk 4]
```diff
@@ -1 +1,174 @@
...
+
+If you are using macOS, please run
+
+```bash
+conda install -c conda-forge niftyreg
+```
+
+in your environment before installing, to ensure all dependencies are installed.
+
+## Usage
```
Comment (adamltyson) [inline README.md:62]:
```suggestion
## Command line usage
```

File: README.md [hunk 5]
```diff
@@ -1 +1,174 @@
...
+```
+
+To find out which atlases are available, once brainreg is installed, please run `brainglobe list`.
+The name of the resulting atlases is the string to pass with the `--atlas` flag.
+
+### Registration backend
+
+To change the registration algorithm used, use the `--backend` flag.
+The default is `niftyreg` as that is currently the only option.
+
```
Comment (adamltyson) [inline README.md:95]: We're not likely to add any more backends to this tool now. 
```suggestion
```

File: README.md [hunk 6]
```diff
@@ -1 +1,174 @@
...
+### Input data orientation
+
+If your data does not match the [brainglobe](https://github.com/brainglobe) default orientation (the origin voxel is the most anterior, superior, left-most voxel), then you must specify the orientation by using the `--orientation` flag.
```
Comment (adamltyson) [inline README.md:98]: This link isn't very informative.

```suggestion
If your data does not match the BrainGlobe default orientation (the origin voxel is the most anterior, superior, left-most voxel), then you must specify the orientation by using the `--orientation` flag.
```

File: README.md [hunk 7]
```diff
@@ -1 +1,174 @@
...
+
+#### Atlas space
+
+You can visualise your data in altas space by enabling the bundled [brainglobe-napari-io](https://github.com/brainglobe/brainglobe-napari-io) plugin:
+
+- Open napari
+- Navigate to `Plugins` -> `Plugin Call Order`
+- In the `Plugin Sorter` window, select `napari_get_reader` from the `select hook...` dropdown box
+- Drag `brainreg_standard` (the atlas space viewer plugin) above `brainreg` (the normal plugin) to ensure that the atlas space plugin is used preferentially.
+
```
Comment (adamltyson) [inline README.md:154]: We no longer support this. 
```suggestion
```

**Review by adamltyson — APPROVED** [review]
Looks great, thanks @willGraham01. I took this opportunity to update some of the text which was out of date in the original repos.

**Comment by willGraham01** [comment]
> I took this opportunity to update some of the text which was out of date in the original repos.

Will keep this in mind when updating the website pages over on `brainglobe.github.io`

**Comment by adamltyson** [comment]
> Will keep this in mind when updating the website pages over on `brainglobe.github.io`

Feel free to raise issues and assign them to me.

**Pull request merged**